addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -205,7 +205,7 @@ def compute_landed_cost(self):
         AdjustementLines = self.env['stock.valuation.adjustment.lines']
         AdjustementLines.search([('cost_id', 'in', self.ids)]).unlink()
 
-        digits = self.env['decimal.precision'].precision_get('Product Price')
+        digits = self.currency_id.decimal_places
```

Comment:
you could call directly self.currency.round() bellow then

Reply:
@amoyaux 
value = tools.float_round(value, precision_digits=digits, rounding_method='UP')
You mean here?

Reply:
No the round of "res.currency" model which call float_round under the hood.